pulp_rpm/app/shared_utils.py

```diff
@@ -2,6 +2,8 @@
 import tempfile
 import shutil
 
+from hashlib import sha256
```

Comment:
@ggainey , @daviddavis , we require sha256, don't we?
So for any configured allowed checksums this should be fine. Please confirm.

Reply:
You're asking in terms of ALLOWED_CONTENT_CHECKSUMS? I think there is the question of whether it's applicable to something like this as you're not dealing with a content artifact. Regardless though, you're correct that sha256 is required.

Reply:
Right. I thought (for whatever reason) that thins configuration is for use in pulp in general and not only for content (it is clear from the name though :D).

Ok, then we are definitely fine and the algorithm doesn't matter if it's not for content artifacts and if it's not md5 or sha1.
Thanks.

Reply:
Concur, sha256 is required for pulp to identify any/everything, so this is def safe - it's the only checksum algorithm whose presence you can rely on.